thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/merge/AnomalyTimeBasedSummarizer.java

```diff
@@ -4,19 +4,14 @@
 import com.linkedin.thirdeye.db.entity.AnomalyResult;
 import java.util.ArrayList;
 import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
 import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.TreeSet;
 
 /**
  * Given list of {@link AnomalyResult} and merge parameters, this utility performs time based merge
  */
-public abstract class AnomalyMergeGenerator {
+public abstract class AnomalyTimeBasedSummarizer {
```

Comment:
Why change the name to summarizer? This is merging, and we're going to have another functionality which will be called summarizer later on